dadi/lib/handlers/css.js

```diff
@@ -24,6 +24,8 @@ const CSSHandler = function (format, req, {
     true
   )
 
+  this.isExternalUrl = this.url.pathname.indexOf('http://') > 0 || this.url.pathname.indexOf('https://') > 0
```

Comment:
Should it be `=== 0`?

Reply:
When parsed an external url in a path looks like this:

```
Url {
        protocol: null,
        slashes: null,
        auth: null,
        host: null,
        port: null,
        hostname: null,
        hash: null,
        search: null,
        query: null,
        pathname: '/http://one.somedomain.tech/test.css',
        path: '/http://one.somedomain.tech/test.css',
        href: '/http://one.somedomain.tech/test.css',
        _raw: '/http://one.somedomain.tech/test.css'
}
```

Reply:
Ah, okay!

Reply:
Do you approve? 😬

Reply:
Yep!